openedx/features/course_experience/urls.py

```diff
@@ -22,6 +23,11 @@
         name='openedx.course_experience.course_updates',
     ),
     url(
+        r'^reviews$',
```

Comment:
It was essentially to do with how complex the code was. Bookmarks was a large Backbone app that already existed, so it made sense for it to be kept as a separate feature. This was the same thing for course search. I could imagine these two features being moved out into their own library at some point. 

For course updates, to me it didn't seem like its own feature. It was just a fragment shown on a very simple page, so I didn't want to introduce all the overhead of a new Django app. I'm happy to revisit that decision, especially if we imagine course updates expanding in scope.